tgs/src/main/cpp/tgs/FeatureSelection/CfsSubsetEvaluator.cpp

```diff
@@ -118,7 +118,7 @@ namespace Tgs
     double classCorrMean = classCorrSum / columns.size();
 
     double intraFactorCorrMean;
-    if (columns.size() == 1)
+    if (columns.size() <= 1)
```

Comment:
If `columns` could get to this point having a size of 0, it is possible to get a divide-by-zero error down below.  Not likely but now no longer possible.